introspection-engine/introspection-engine-tests/tests/simple.rs

```diff
@@ -80,16 +80,29 @@ Example usage:
 source .test_database_urls/mysql_5_6
     "#);
 
+    if database_url == "sqlite" {
+        database_url = format!("{}.db", sqlite_test_url(test_function_name));
+
+        let file = database_url.trim_start_matches("file:");
+        std::fs::remove_file(&file).ok();
+    }
```

Comment:
I think there's `sqlite_test_file()` in `test_setup` that does the sqlite file truncation too.

Reply:
Actually, I'm already using this through `sqlite_test_url`. It somehow uses the same file in the subsequent runs... Therefore removing it here.